src/System.IO.UnmanagedMemoryStream/ref/project.json

```diff
@@ -2,8 +2,8 @@
   "dependencies": {
     "System.IO.FileSystem.Primitives": "4.0.1",
     "System.IO": "4.1.0",
-    "System.Runtime.InteropServices": "4.1.0",
-    "System.Runtime": "4.1.0",
+    "System.Runtime.InteropServices": "4.4.0-beta-24608-01",
```

Comment:
Once you move the other type the only 2 references that should be here are System.Runtime and System.Runtime.InteropServies.